botocore/useragent.py

```diff
@@ -116,6 +116,14 @@ def to_string(self):
         return self._value
 
 
+try:
+    from botocore.customizations.useragent import modify_components
```

Comment:
Can we add a disclaimer here to effect of:
```
# This is not a public interface and subject to abrupt breaking changes.
# Any usage is not advised or supported in external code bases.

Reply:
Added in latest changes.